src/components/header/Header.tsx

```diff
@@ -10,12 +10,14 @@ import {
   DialogContent,
   DialogContentText,
   DialogTitle,
+  Divider,
```

Comment:
changing the MUI4 components with the MUI5 versions provoke some undesired changes in the header, namely
- the centered title changes color, it is black, it should be white.
- the icons right-hand side gets more closer than expected.